client/src/ducks/matchable.js

```diff
@@ -7,5 +8,9 @@ export const matchGroup = (maxMembers, courseID, date, hasTime) => {
 
 export const createGroup = (startDate, courseID, groupSize, duration, userID) => {
   const body = { groupSize, startDate, courseID, duration, userID };
-  return axios.post('/api/matchable/groups', body);
+  return withNotification(
+    'Group created successfully',
```

Comment:
make the sentences constants inside ducks/matchable/constants, there should not be any hardcoded strings inside function calls